packages/hash/api/src/model/knowledge/entity.model.ts

```diff
@@ -284,13 +283,11 @@ export default class {
   /**
    * Get the latest version of an entity by its entity ID.
    *
-   * @param params.accountId the accountId of the account requesting the entity
    * @param params.versionedUri the unique versioned URI for an entity.
    */
   static async getLatest(
     graphApi: GraphApi,
     params: {
-      accountId?: string;
```

Comment:
I've removed `accountId` from methods where it is currently not being used. Once we start sharding on `ownedById`, the field should be re-introduced. The type errors will remind us to do this, so I didn't think it was worth adding a todo